runtime/runtime-params-estimator/src/rocksdb.rs

```diff
@@ -0,0 +1,311 @@
+use std::{io::prelude::*, path::PathBuf};
+
+use clap::Clap;
+use rand::{prelude::SliceRandom, Rng};
+use rand_xorshift::XorShiftRng;
+use rocksdb::DB;
+
+use crate::{config::Config, gas_cost::GasCost};
+
+#[derive(Debug, Clone, Clap)]
+pub struct RocksDBTestConfig {
+    /// Value size used for all DB operations in RocksDB tests
+    /// (`RocksDb*` estimations only)
+    #[clap(name = "rdb-value-size", long, default_value = "1000")]
+    pub value_size: usize,
+    /// Number of insertions/reads performed in RocksDB tests
+    /// (`RocksDb*` estimations only)
+    #[clap(name = "rdb-op-count", long, default_value = "1000000")]
+    pub op_count: usize,
+    /// Size of memtable used in RocksDB tests
+    /// (`RocksDb*` estimations only)
+    #[clap(name = "rdb-memtable-size", long, default_value = "256000000")]
+    pub memtable_size: usize,
+    /// Number of insertions into test DB before measurement begins
+    /// (`RocksDb*` estimations only)
+    #[clap(name = "rdb-setup-insertions", long, default_value = "2000000")]
+    pub setup_insertions: usize,
+    /// Keys will be ordered sequentially if this is set, randomized otherwise.
+    /// (`RocksDb*` estimations only)
+    #[clap(name = "rdb-sequential-keys", long)]
+    pub sequential_keys: bool,
+    /// Flush after a bulk of write operations.
+    /// The flush time will be included in the reported measurement.
+    /// (`RocksDb*` estimations only)
+    #[clap(long, name = "rdb-force-flush", long)]
+    pub force_flush: bool,
+    /// Force compactions after a bulk of operations.
+    /// The compaction time will be included in the reported measurement.
+    /// (`RocksDb*` estimations only)
+    #[clap(long, name = "rdb-force-compaction", long)]
+    pub force_compaction: bool,
+    /// Enable the default block cache used for reads, disabled by default.
+    /// (`RocksDb*` estimations only)
+    #[clap(long, name = "rdb-block-cache", long)]
+    pub block_cache: bool,
+    /// Print RocksDB debug output where available
+    #[clap(skip)]
+    pub debug_rocksdb: bool,
+    /// Pseudo-random input data dump
+    /// (`RocksDb*` estimations only)
+    #[clap(long, name = "rdb-pr-data-path", long)]
+    pub pr_data_path: Option<PathBuf>,
```

Comment:
nit: this dosen't *have* to be PR, so maybe a better name is possible for the flag. Not that it matters. 

pr is also not the most immediatelly obvious prefix.

Reply:
Changes this to `rdb-input-data-path` which personally I like much better but it is still not perfect (names are hard)